### PR TITLE
Import Expo.fx in AppEntry

### PR DESCRIPTION
**Pull request by brentvatne**
# Why

We changed AppEntry.js to import registerRootComponent directly rather than via the expo package to improve reload performance, but this had the effect of no longer loading fx until the expo package is imported by the user. This adds back importing fx but does it directly rather than via Expo package.

# How

Chat with @EvanBacon and @ide about the preferred way to handle it. Considered customizing metro serializer but this was much lower touch and works on any bundler.

# Test Plan

Run an app that doesn't import from the `expo` package in App.js, logging etc should work.

### CODE REVIEW

File: packages/expo/AppEntry.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import 'expo/build/Expo.fx';
```
Comment (EvanBacon) [inline packages/expo/AppEntry.js]: We should create a doc in the future which describes how to create a custom main entry point.

**Review by EvanBacon — APPROVED** [review]
LGTM

**Review by ide — APPROVED** [review]

File: packages/expo/src/Expo.fx.ts [hunk 2]
```diff
@@ -7,9 +7,9 @@ import 'expo-asset';
 
 import { AppRegistry, Platform } from 'react-native';
 import Constants from 'expo-constants';
+import DevAppContainer from './environment/DevAppContainer';
```
[the remark this answers is not in the document]
Comment (brentvatne) [reply packages/expo/src/Expo.fx.ts]: nope. just ran `yarn lint --fix` and it's in a new spot now so i'll leave it there

File: packages/expo/src/Expo.ts [hunk 3]
```diff
@@ -161,19 +161,4 @@ export {
...
-
-    // @ts-ignore
-    global.__exponent = globals;
-    // @ts-ignore
-    global.__expo = globals;
-    // @ts-ignore
-    global.Expo = globals;
-  }
-}
+} from './removed';
```
[the remark this answers is not in the document]
Comment (brentvatne) [reply packages/expo/src/Expo.ts]: `yarn lint --fix` and it fixed this

**Pull request merged**